Mathlib/Order/CompleteBooleanAlgebra.lean

```diff
@@ -60,26 +61,32 @@ except for implementing `Order.Frame` via `Order.Frame.ofMinimalAxioms`.
 This structure omits the `himp`, `compl` fields, which can be recovered using
 `Order.Frame.ofMinimalAxioms`. -/
 class Order.Frame.MinimalAxioms (α : Type u) extends CompleteLattice α where
-  inf_sSup_le_iSup_inf (a : α) (s : Set α) : a ⊓ sSup s ≤ ⨆ b ∈ s, a ⊓ b
+  protected inf_sSup_le_iSup_inf (a : α) (s : Set α) : a ⊓ sSup s ≤ ⨆ b ∈ s, a ⊓ b
 
 /-- Structure containing the minimal axioms required to check that an order is a coframe. Do NOT
 use, except for implementing `Order.Coframe` via `Order.Coframe.ofMinimalAxioms`.
 
 This structure omits the `sdiff`, `hnot` fields, which can be recovered using
 `Order.Coframe.ofMinimalAxioms`. -/
 class Order.Coframe.MinimalAxioms (α : Type u) extends CompleteLattice α where
-  iInf_sup_le_sup_sInf (a : α) (s : Set α) : ⨅ b ∈ s, a ⊔ b ≤ a ⊔ sInf s
+  protected iInf_sup_le_sup_sInf (a : α) (s : Set α) : ⨅ b ∈ s, a ⊔ b ≤ a ⊔ sInf s
```

Comment:
They don't need to be protected anymore since you removed the lemma that was in the root namespace